graylog2-web-interface/src/components/lookup-tables/DataAdapter.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { PluginStore } from 'graylog-web-plugin/plugin';
+import type { PluginExports } from 'graylog-web-plugin/plugin';
+
+import { LinkContainer } from 'components/common/router';
+import { Row, Col, Button, Input } from 'components/bootstrap';
+import { getValueFromInput } from 'util/FormsUtils';
+import Routes from 'routing/Routes';
+import { LookupTableDataAdaptersActions } from 'stores/lookup-tables/LookupTableDataAdaptersStore';
+import type { LookupTableAdapter } from 'logic/lookup-tables/types';
+import useScopePermissions from 'hooks/useScopePermissions';
+
+import ConfigSummaryDefinitionListWrapper from './ConfigSummaryDefinitionListWrapper';
+
+type Props = {
+  dataAdapter: LookupTableAdapter,
+};
+
+const DataAdapter = ({ dataAdapter }: Props) => {
+  const [lookupKey, setLookupKey] = React.useState(null);
+  const [lookupResult, setLookupResult] = React.useState(null);
+  const { loadingScopePermissions, scopePermissions } = useScopePermissions(dataAdapter);
+
+  const _onChange = (event) => {
```

Comment:
You did it in most cases, but we try to always type the function parameters, if they can not be inferred. Current `event` is implicitly `any`.

By typing it we can activate the TS config option `noImplicitAny` at some point.